website_sale_qty/models/product_template.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import date
+from openerp import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    price_quantity_tiers = fields.Serialized(
+        compute='_compute_price_quantity_tiers',
+        help='Unit prices by pricelist and minimum quantity',
+    )
+
+    @api.multi
+    def _compute_price_quantity_tiers(self):
+        pricelists = self.env['product.pricelist'].search([('id', '>', 0)])
+
+        for record in self:
+            results = {}
+
+            for pricelist in pricelists:
+                pricelist_items = self.env['product.pricelist.item'].search([
+                    ('pricelist_id', '=', pricelist.id),
+                    ('product_tmpl_id', '=', record.id),
+                    '|',
+                    ('date_start', '<=', date.today()),
```

Comment:
You're putting an object in a domain. That's invalid. It should be a string in the proper format, which is what fields does.